code/modules/events/ion_storm.dm

```diff
@@ -566,6 +566,35 @@
 							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
 
 	return message
+	
+
+/proc/corp_name()

```

Comment:
no, I told him to use the pick_lists system like the AI ion laws uses and of course he didn't take the time to understand what I actually meant.